src/main/java/redlightBack/Comment/CommentController.java

```diff
@@ -0,0 +1,80 @@
+package redlightBack.Comment;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.*;
+import redlightBack.Comment.Dto.CommentLikeResponse;
+import redlightBack.Comment.Dto.CommentResponse;
+import redlightBack.Comment.Dto.CommentResponseAndPaging;
+import redlightBack.Comment.Dto.CreateCommentRequest;
+import redlightBack.loginUtils.LoginMemberId;
+
+import java.nio.file.AccessDeniedException;
+
+
+@RequiredArgsConstructor
+@RestController
+public class CommentController {
+
+    public final CommentService commentService;
+
+
+    @PostMapping("/comments")
+    public CommentResponse createComment(@LoginMemberId String userId,
+                                         @RequestBody CreateCommentRequest request) {
+        return commentService.save(userId, request);
+    }
+
+    //댓글+대댓글 조회
+//    @GetMapping("/comments")
+//    public CommentResponseAndPaging getCommentTree(@RequestParam Long postId,
+//                                                   @RequestParam(defaultValue = "1") int size,
+//                                                   @RequestParam(defaultValue = "150") int pageNumber ){
+//
+//        Pageable pageable = PageRequest.of(pageNumber -1, size);
+//
+//        return commentService.getCommentTree(postId, pageable);
+//    }
+
+
+    //댓글 수정 로그인 유저만
+    @PutMapping("/comments/{commentId}")
+    public void updateComment(@LoginMemberId String userId,
+                              @PathVariable Long commentId,
+                              @RequestBody CreateCommentRequest request) throws AccessDeniedException {
+        commentService.updateComment(request, userId, commentId);
+    }
+
+
+    //소프트삭제
+    @DeleteMapping("/{commentId}")
+    public void deleteComment(@LoginMemberId String userId,
+                              @PathVariable Long commentId) throws AccessDeniedException {
```

Comment:
- 컨트롤러는 클라이언트에게 제공되는 API를 정의하는 곳이고, API를 이용하는 클라이언트 입장에서는 soft delete든 hard delete든 관심이 없고 삭제인 것만 중요합니다.
- 수정 endpoint와 일관성 있게, 또 RESTful하게 URL을 아래처럼 고치는 것이 좋습니다.
```suggestion
    // 삭제
    @DeleteMapping("/comments/{commentId}")
    public void deleteComment(@LoginMemberId String userId,
                              @PathVariable Long commentId) throws AccessDeniedException {
```

Reply:
RequestMapping 추가햇습니다.

Reply:
기존 코드와의 일관성을 고려해서 `comments` 왼쪽에 `/`까지 붙이는 것이 좋을 것 같아요.